src/styles/styles.js

```diff
@@ -696,11 +697,13 @@ const styles = {
             textAlignVertical: 'center',
         },
         textInput: {
+            padding: 0,
             outlineStyle: 'none',
             color: themeColors.text,
             fontFamily: fontFamily.GTA,
             fontSize: variables.fontSizeNormal,
             textAlignVertical: 'center',
+            lineHeight: (Platform.OS === 'ios') ? 19 : 20,
```

Comment:
Why do we need to use different line-heights for IOS and android? 

I believe our box model has some issues which are causing these misalignments. If this is an issue with RN native code then I would prefer it to be fixed there. 

I would like to see substantial proof for this change or a valid argument why do we have to do this change?

Reply:
Just noting that if we would want to debug this further and find where this issue is caused in RN, this would be out of scope for this issue/job. Since it would most likely take more time.